dffml/util/cli/cmds.py

```diff
@@ -70,31 +72,51 @@ def __init__(self, *args, **kwargs):
                 self.sources[i] = self.sources[i].withconfig(self.extra_config)
 
 
+
+@config
+class ModelCMDConfig(CMDConfig):
+    model: Model = field(
+        "Model used for ML",
+        default_factory = Model.load,
+    )
+
+
 class ModelCMD(CMD):
     """
     Set a models model dir.
     """
 
-    arg_model = Arg(
-        "-model", help="Model used for ML", type=Model.load, required=True
-    )
-
+    CONFIG = ModelCMDConfig
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if inspect.isclass(self.model):
             self.model = self.model.withconfig(self.extra_config)
 
 
+@config
+class PortCMDConfig(CMDConfig):
+    port: Port = field(
+        "Port",
+        default_factory = Port.load,
+    )
+
+
 class PortCMD(CMD):
 
-    arg_port = Arg("port", type=Port.load)
+    CONFIG = PortCMDConfig
+
+
+@config
+class KeysCMDConfig(CMDConfig):
+    keys: Port = field(
```

Comment:
```suggestion
    keys: str = field(
```

Reply:
Oops! This must've slipped by while copy pasting. Updated! :D